src/main/java/seedu/mark/model/reminder/Reminder.java

```diff
@@ -0,0 +1,58 @@
+package seedu.mark.model.reminder;
+
+import static seedu.mark.commons.util.CollectionUtil.requireAllNonNull;
+
+import java.util.Date;
```

Comment:
Can we use `java.time.LocalDateTime` instead? Many of [`java.util.Date`](https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/util/Date.html)'s methods seem to be marked as **Deprecated**.

Reply:
Okay.